src/elements/common/pagination/__tests__/MarkerBasedPagination.test.tsx

```
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '../../../../test-utils/testing-library';
+import MarkerBasedPagination, { MarkerBasedPaginationProps } from '../MarkerBasedPagination';
+
+describe('elements/common/pagination/MarkerBasedPagination', () => {
+    const renderComponent = (props: Partial<MarkerBasedPaginationProps>) =>
+        render(<MarkerBasedPagination isSmall={false} {...props} />);
+
+    test('should not render pagination controls when both markers are false', () => {
+        renderComponent({
+            hasNextMarker: false,
+            hasPrevMarker: false,
```

do we need to set it to false, isnt it already false by default?

---

it cant tell whether it is set to false by default from the test file, so I choose to pass it as false here just to make it more clear.

---

do we need to know that by default that you need to set both to false to hide it, cant we just update the title to say that instead?

---

updating test title makes sense to me